include/sunlinsol/sunlinsol_ginkgo.hpp

```diff
@@ -332,7 +332,7 @@ class LinearSolver : public ConvertibleTo<SUNLinearSolver>
     }
 
     iter_count_ = static_cast<int>(logger->get_num_iterations());
-    GkoExec()->get_master()->copy_from(gko::lend(GkoExec()), 1,
+    GkoExec()->get_master()->copy_from(GkoExec(), 1,
```

Comment:
`gko::lend` is deprecated but I'm not sure what version in which it was deprecated. We probably need a version specific macro for backward compatibility.

Reply:
It appears that it was deprecated in 1.7.0

Reply:
Turns out it's actually 1.6 where `gko::lend` was deprecated. The change log notes it more clearly in 1.7 but it's also mentioned in the 1.6 change log. Tested with 1.5 and 1.6 to confirm.